src/server/routes/proxy.route.ts

```diff
@@ -49,6 +49,128 @@ export const embeddings: MiddlewareHandler<Env> = async c => {
   return proxyOpenAI(c, result, provider, '/v1/embeddings')
 }
 
+export const responses: MiddlewareHandler<Env> = async c => {
+  const { result, provider } = await processRequest(c)
+  return proxyOpenAI(c, result, provider, '/v1/responses')
+}
+
+export const realtimeClientSecrets: MiddlewareHandler<Env> = async c => {
+  const result = await c.req.valid('json')
+  const sessionType = (result as any).session?.type
+  const model = (result as any).session?.model
+
+  if (sessionType === 'transcription') {
```

Comment:
this if feels unecessary, not sure we need to support not including the model in the request